packages/frontend/src/notebook/notebook_editor.tsx

```diff
@@ -36,6 +37,260 @@ import {
 
 import "./notebook_editor.css";
 
+export function WalkthroughOverlay(props: { isOpen: boolean; onClose: () => void }) {
```

Comment:
I think this component should get moved into it's own file. Both the walkthrough the notebook editor are going to get a lot bigger before this project is over, also their implementation seems almost entirely orthogonal.